src/test/java/org/apache/commons/dbutils/BeanProcessorTest.java

```diff
@@ -186,19 +187,21 @@ public void setTestField(final int idx, final Integer testField) {
         }
     }
 
-    private static final BeanProcessor beanProc = new BeanProcessor();
+    private static final BeanProcessor BEAN_PROCESSOR = new BeanProcessor();
 
     public void testCheckAnnotationOnMissingReadMethod() throws Exception {
         final String[] colNames = { "testField" };
         final ResultSetMetaData metaData = MockResultSetMetaData.create(colNames);
 
-        final String testField = "first";
-        final Object[][] rows = { new Object[] { testField } };
+        final String testFieldRowValues = "first";
+        final Object[][] rows = {
```

Comment:
Don't reformat the code, it makes the PR noisy and takes longer to review.

Reply:
Sure, Fixed it @garydgregory